docs/src/main/paradox/java/http/routing-dsl/index.md

```diff
@@ -1,6 +1,6 @@
 # High-level Server-Side API
 
-In addition to the @ref[Low-Level Server-Side API](../server-side/low-level-api.md) Akka HTTP provides a very flexible "Routing DSL" for elegantly
+In addition to the @ref[Low-Level Server-Side API](../../../scala/http/server-side/low-level-api.md) Akka HTTP provides a very flexible "Routing DSL" for elegantly
```

Comment:
This will break the unified Java/Scala docs. We need to keep links as relative as possible.

Reply:
This commit should be dropped completely it seems.